vcpkg/concepts/system-package-managers.md

```diff
@@ -0,0 +1,23 @@
+---
+title: System package managers
+description: Concepts on system package managers
+author: data-queue
+ms.author: danshaw2
+ms.date: 01/10/2024
+ms.prod: vcpkg
+ms.topic: concept-article
+---
+
+# System package managers
+
+C++ libraries are available in system package managers, such as apt, and vcpkg. How you choose to acquire your libraries depends on your specific usecase.
```

Comment:
```suggestion
C++ libraries are available in system package managers such as apt and vcpkg. How you choose to acquire your libraries depends on your specific use case.
```

Reply:
```suggestion
C++ libraries are available in system package managers, such as apt, and in vcpkg. How you choose to acquire your libraries depends on your specific use case.
```
I initially read "such as apt and vcpkg", but this isn't what is meant...